docs/excel/excel-add-ins-pivottables.md

```diff
@@ -221,7 +222,142 @@ Excel.run(function (context) {
 });
 ```
 
-## Slicers
+## Filter a PivotTable
+
+The primary method for filtering PivotTable data is with PivotFilters. Slicers offer an alternate, less flexible filtering method. 
+
+[PivotFilters](/javascript/api/excel/excel.pivotfilters) filter data based on a PivotTable's four [hierarchy categories](#hierarchies) (filters, columns, rows, and values). There are four types of PivotFilters, allowing calendar date-based filtering, string parsing, number comparison, and filtering based on a custom input. 
```

Comment:
Might be worth having a line that reconciles PivotFilters and FilterPivotHierarchy. Perhaps in the hierarchy section? This could be done as a separate PR (so you can look at this with fresh eyes.

Reply:
Internal tracking info: [work item 4697638](https://office.visualstudio.com/OC/_workitems/edit/4697638)